### PR TITLE
UI tests, testing renaming a file/folder into .htaccess

### PR DESCRIPTION
**Pull request by individual-it**
some additional UI tests

### CODE REVIEW

File: tests/ui/features/files/renameFiles.feature [hunk 1]
```diff
@@ -45,6 +45,13 @@ Feature: renameFiles
 		|Could not rename "data.zip"|
 		And the file "data.zip" should be listed
 
+	Scenario: Rename a file to a forbidden name
```
Comment (phil-davis) [inline tests/ui/features/files/renameFiles.feature]: Isn't this the same as the .htaccess case of the scenario at line 37?

File: tests/ui/features/files/renameFolders.feature [hunk 2]
```diff
@@ -44,6 +44,13 @@ Feature: renameFolders
 		|Could not rename "simple-folder"|
 		And the folder "simple-folder" should be listed
 
+	Scenario: Rename a folder to a forbidden name
```
Comment (phil-davis) [inline tests/ui/features/files/renameFolders.feature]: and this is already on the scenario at line 36?

**Comment by individual-it** [comment]
true deleted the duplication. Maybe its good to keep the separate https://github.com/owncloud/core/pull/28737#issuecomment-323309530
If not merge the ones in master too

**Comment by phil-davis** [comment]
So do you just want to make a separate scenario so that the scenario text is more accurate?
Because splitting this out will take a bit more run time.

Also, if you do it this way, you need to also delete the extra 'Could not rename "data.zip"' in the other scenarios.

**Comment by individual-it** [comment]
> So do you just want to make a separate scenario so that the scenario text is more accurate?
Because splitting this out will take a bit more run time.

ether splitting here or merging in master

**Pull request merged**